### ISSUE TITLE
虚拟环境你提供的版本不对

### ISSUE THREAD

**Issue by ikko-debug** [wzhwzhwzh0921/S-D-Mamba#33]
请提供完整的requirements.txt谢谢

**Comment by wellhaomin** [comment]
请问解决了吗

**Comment by ikko-debug** [comment]
name: smamba
channels:
  - nvidia
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/pytorch
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/r
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free/
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main/
  - pytorch
  - defaults
dependencies:
  - _libgcc_mutex=0.1=main
  - _openmp_mutex=5.1=1_gnu
  - backcall=0.2.0=pyhd3eb1b0_0
  - blas=1.0=mkl
  - brotlipy=0.7.0=py38h27cfd23_1003
  - bzip2=1.0.8=h7b6447c_0
  - ca-certificates=2023.01.10=h06a4308_0
  - certifi=2023.5.7=py38h06a4308_0
  - cffi=1.15.1=py38h5eee18b_3
  - charset-normalizer=2.0.4=pyhd3eb1b0_0
  - cryptography=39.0.1=py38h9ce1e76_0
  - cuda-cudart=11.7.99=0
  - cuda-cupti=11.7.101=0
  - cuda-libraries=11.7.1=0
  - cuda-nvrtc=11.7.99=0
  - cuda-nvtx=11.7.91=0
  - cuda-runtime=11.7.1=0
  - decorator=5.1.1=pyhd3eb1b0_0
  - ffmpeg=4.3=hf484d3e_0
  - filelock=3.9.0=py38h06a4308_0
  - freetype=2.12.1=h4a9f257_0
  - giflib=5.2.1=h5eee18b_3
  - gmp=6.2.1=h295c915_3
  - gmpy2=2.1.2=py38heeb90bb_0
  - gnutls=3.6.15=he1e5248_0
  - idna=3.4=py38h06a4308_0
  - importlib-metadata=6.0.0=py38h06a4308_0
  - importlib_metadata=6.0.0=hd3eb1b0_0
  - intel-openmp=2023.1.0=hdb19cb5_46305
  - jinja2=3.1.2=py38h06a4308_0
  - jpeg=9e=h5eee18b_1
  - jupyter_client=8.1.0=py38h06a4308_0
  - jupyter_core=5.3.0=py38h06a4308_0
  - lame=3.100=h7b6447c_0
  - lcms2=2.12=h3be6417_0
  - ld_impl_linux-64=2.38=h1181459_1
  - lerc=3.0=h295c915_0
  - libcublas=11.10.3.66=0
  - libcufft=10.7.2.124=h4fbf590_0
  - libcufile=1.6.1.9=0
  - libcurand=10.3.2.106=0
  - libcusolver=11.4.0.1=0
  - libcusparse=11.7.4.91=0
  - libdeflate=1.17=h5eee18b_0
  - libffi=3.4.4=h6a678d5_0
  - libgcc-ng=11.2.0=h1234567_1
  - libgomp=11.2.0=h1234567_1
  - libiconv=1.16=h7f8727e_2
  - libidn2=2.3.4=h5eee18b_0
  - libnpp=11.7.4.75=0
  - libnvjpeg=11.8.0.2=0
  - libpng=1.6.39=h5eee18b_0
  - libsodium=1.0.18=h7b6447c_0
  - libstdcxx-ng=11.2.0=h1234567_1
  - libtasn1=4.19.0=h5eee18b_0
  - libtiff=4.5.0=h6a678d5_2
  - libunistring=0.9.10=h27cfd23_0
  - libwebp=1.2.4=h11a3e52_1
  - libwebp-base=1.2.4=h5eee18b_1
  - lz4-c=1.9.4=h6a678d5_0
  - markupsafe=2.1.1=py38h7f8727e_0
  - matplotlib-inline=0.1.6=py38h06a4308_0
  - mkl=2023.1.0=h6d00ec8_46342
  - mkl-service=2.4.0=py38h5eee18b_1
  - mkl_fft=1.3.6=py38h417a72b_1
  - mkl_random=1.2.2=py38h417a72b_1
  - mpc=1.1.0=h10f8cd9_1
  - mpfr=4.0.2=hb69a4c5_1
  - mpmath=1.2.1=py38h06a4308_0
  - ncurses=6.4=h6a678d5_0
  - nest-asyncio=1.5.6=py38h06a4308_0
  - nettle=3.7.3=hbbd107a_1
  - networkx=2.8.4=py38h06a4308_1
  - numpy=1.24.3=py38hf6e8229_1
  - numpy-base=1.24.3=py38h060ed82_1
  - openh264=2.1.1=h4ff587b_0
  - openssl=1.1.1t=h7f8727e_0
  - packaging=23.0=py38h06a4308_0
  - parso=0.8.3=pyhd3eb1b0_0
  - pexpect=4.8.0=pyhd3eb1b0_3
  - pickleshare=0.7.5=pyhd3eb1b0_1003
  - pillow=9.4.0=py38h6a678d5_0
  - pip=23.0.1=py38h06a4308_0
  - ptyprocess=0.7.0=pyhd3eb1b0_2
  - pure_eval=0.2.2=pyhd3eb1b0_0
  - pycparser=2.21=pyhd3eb1b0_0
  - pygments=2.15.1=py38h06a4308_1
  - pyopenssl=23.0.0=py38h06a4308_0
  - pysocks=1.7.1=py38h06a4308_0
  - python=3.8.16=h7a1cb2a_3
  - python-dateutil=2.8.2=pyhd3eb1b0_0
  - pytorch-cuda=11.7=h778d358_5
  - pytorch-mutex=1.0=cuda
  - readline=8.2=h5eee18b_0
  - requests=2.29.0=py38h06a4308_0
  - setuptools=66.0.0=py38h06a4308_0
  - six=1.16.0=pyhd3eb1b0_1
  - sqlite=3.41.2=h5eee18b_0
  - stack_data=0.2.0=pyhd3eb1b0_0
  - sympy=1.11.1=py38h06a4308_0
  - tbb=2021.8.0=hdb19cb5_0
  - tk=8.6.12=h1ccaba5_0
  - torchaudio=2.0.2=py38_cu117
  - torchvision=0.15.2=py38_cu117
  - urllib3=1.26.15=py38h06a4308_0
  - wheel=0.38.4=py38h06a4308_0
  - xz=5.4.2=h5eee18b_0
  - zeromq=4.3.4=h2531618_0
  - zipp=3.11.0=py38h06a4308_0
  - zlib=1.2.13=h5eee18b_0
  - zstd=1.5.5=hc292b87_0
  - pip:
      - anyio==3.7.0
      - argon2-cffi==21.3.0
      - argon2-cffi-bindings==21.2.0
      - argparse==1.4.0
      - arrow==1.2.3
      - asttokens==2.2.1
      - attrs==23.1.0
      - automat==24.8.1
      - axial-positional-embedding==0.3.12
      - beautifulsoup4==4.12.2
      - bleach==6.0.0
      - buildtools==1.0.6
      - calmsize==0.1.3
      - causal-conv1d==1.1.0
      - colt5-attention==0.11.1
      - comm==0.1.3
      - constantly==23.10.4
      - contourpy==1.1.1
      - cycler==0.12.1
      - debugpy==1.6.7
      - defusedxml==0.7.1
      - deprecated==1.2.18
      - docopt==0.6.2
      - einops==0.8.1
      - exceptiongroup==1.1.1
      - executing==1.2.0
      - fastjsonschema==2.17.1
      - fonttools==4.57.0
      - fqdn==1.5.1
      - fsspec==2024.6.1
      - furl==2.1.4
      - greenlet==3.1.1
      - hf-xet==1.1.5
      - huggingface-hub==0.34.1
      - hyperlink==21.0.0
      - importlib-resources==5.12.0
      - incremental==24.7.2
      - ipykernel==6.23.1
      - ipython==8.12.2
      - ipython-genutils==0.2.0
      - ipywidgets==8.0.6
      - isoduration==20.11.0
      - jedi==0.18.2
      - joblib==1.4.2
      - jsonpointer==2.3
      - jsonschema==4.17.3
      - jupyter==1.0.0
      - jupyter-client==8.2.0
      - jupyter-console==6.6.3
      - jupyter-events==0.6.3
      - jupyter-server==2.6.0
      - jupyter-server-terminals==0.4.4
      - jupyterlab-pygments==0.2.2
      - jupyterlab-widgets==3.0.7
      - kiwisolver==1.4.7
      - llvmlite==0.41.1
      - local-attention==1.10.0
      - mamba-ssm==1.1.0
      - matplotlib==3.7.5
      - mistune==2.0.5
      - nbclassic==1.0.0
      - nbclient==0.8.0
      - nbconvert==7.4.0
      - nbformat==5.9.0
      - ninja==1.11.1.4
      - notebook==6.5.4
      - notebook-shim==0.2.3
      - numba==0.58.1
      - nvidia-ml-py==12.575.51
      - nvitop==1.5.2
      - orderedmultidict==1.0.1
      - overrides==7.3.1
      - pandas==1.5.3
      - pandocfilters==1.5.0
      - patool==1.12
      - pkgutil-resolve-name==1.3.10
      - platformdirs==3.5.1
      - product-key-memory==0.2.11
      - prometheus-client==0.17.0
      - prompt-toolkit==3.0.38
      - psutil==5.9.5
      - pyparsing==3.1.4
      - pyrsistent==0.19.3
      - python-json-logger==2.0.7
      - pytorch-memlab==0.3.0
      - pytz==2023.3.post1
      - pywavelets==1.4.1
      - pyyaml==6.0
      - pyzmq==25.1.0
      - qtconsole==5.4.3
      - qtpy==2.3.1
      - redo==3.0.0
      - reformer-pytorch==1.4.4
      - regex==2024.11.6
      - rfc3339-validator==0.1.4
      - rfc3986-validator==0.1.1
      - safetensors==0.5.3
      - scikit-learn==1.2.2
      - scipy==1.10.1
      - send2trash==1.8.2
      - simplejson==3.20.1
      - sktime==0.16.1
      - sniffio==1.3.0
      - soupsieve==2.4.1
      - sqlalchemy==2.0.41
      - stack-data==0.6.2
      - terminado==0.17.1
      - thop==0.1.1-2209072238
      - threadpoolctl==3.5.0
      - tinycss2==1.2.1
      - tokenizers==0.20.3
      - tomli==2.2.1
      - torch==2.1.2+cu121
      - tornado==6.3.2
      - tqdm==4.67.1
      - traitlets==5.9.0
      - transformers==4.46.3
      - triton==2.1.0
      - twisted==25.5.0
      - typing-extensions==4.13.2
      - tzdata==2023.3
      - uri-template==1.2.0
      - wcwidth==0.2.6
      - webcolors==1.13
      - webencodings==0.5.1
      - websocket-client==1.5.2
      - widgetsnbextension==4.0.7
      - wrapt==1.17.3
      - zope-interface==7.2
prefix: /home/share/HDstorage/xyc/envs/smamba
